docs/pywbemcli/generaloptions.rst

```diff
@@ -393,13 +393,24 @@ any subsequent pywbemcli commands:
 ``--timeout`` general option
 """"""""""""""""""""""""""""
 
-The argument value of the ``--timeout``/``-t`` general option is an integer that
-defines the
-client side timeout in seconds. The pywbem client includes a timeout mechanism
-that closes a WBEM connection if there is no response to a request to the WBEM
-server in the time defined by this value. Pywbemcli defaults to a
+The argument value of the ``--timeout``/``-t`` general option is an integer
+that defines the client side read timeout in seconds. The pywbem client
+includes a timeout mechanism that closes a WBEM connection if there is no
+response to a request to the WBEM server in the time defined by this value and
+the retry count.
+
+The option value is the timeout for a single read request to the WBEM server. Since
+pywbem retries the command multiple times (normally 2 retries), the time to
+command failure may be multiple times this timeout value.  Pywbemcli defaults to a
 predefined timeout (normally 30 seconds) if this option is not defined.
 
+The connection functionality has a separate timeout defined by pywbem and normally
+set at 10 seconds also with retries.
```

Comment:
I think we have constants for the connection timeout and its retries, and I suggest that we change the text to be specific about that.

Reply:
Same comment as above to avoid specific values that are controlled by variables to which the documents do not have direct access.